airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -164,13 +164,11 @@ Translation conflicts MUST be resolved according to the procedures outlined in s
 ### 6.1. Approval of ownership candidates
 
 - The designated code owner, should post a thread to the dev list, requesting the approval of:
-  - Themselves as the code owner in the suggested locale. If they are a translation sponsor, they should indicate it as well.
-  - Other code owner(s) in the suggested locale, if applicable.
-  - The identity of the translation owner(s) in the suggested locale (which could be themselves as well)
```

Comment:
Everything related to 'code owners' shouldn't go trough the dev list because only committer can be code owners.